cms/static/sass/views/_video-upload.scss

```diff
@@ -299,10 +280,24 @@
       }
     }
 
-    &.edit .thumbnail-action {
-      background-color: white;
+    &.edit {
+      background-color: #4e4e4e;
```

Comment:
@muzaffaryousaf I coudn't find this color in variables. So if ti's used once, I think we should be OK?